test/functional/NativeTest/playlist.xml

```diff
@@ -28,7 +28,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 			<variation>NoOptions</variation>
 		</variations>
 		<command>$(ADD_JVM_LIB_DIR_TO_LIBPATH) \
-	$(SQ)$(JAVA_SHARED_LIBRARIES_DIR)$(D)algotest$(SQ) -avltest:$(TEST_RESROOT)$(D)..$(D)algotest$(D)avltest.lst -Xcheck:memory:ignoreUnfreedCallsite=zip/:dbgwrapper:unknown:wrapper; \
```

Comment:
The uses of `$(SQ)` were significant, they prevented interpretation of backslashes in paths - they need to be restored.
```
/usr/bin/sh: line 4: w:buildsbld_99009/sdk/wa6480jrebindefaultvmtest: No such file or directory
```
That path should be
```
w:\builds\bld_99009/sdk/wa6480\jre\bin\default\vmtest
```